src/page/AppLayout.js

```diff
@@ -1,25 +1,35 @@
 import {Outlet} from "react-router-dom";
 
 import {getAuthTokenCookie} from "../tool/auth.util";
+import {NAVIGATION_ITEMS, ROUTE} from "../tool/constant";
+import {NavigationContext} from "../context/Navigation.context";
 
 import PageHeader from "../component/page/PageHeader";
 import LoginContainer from "../container/Login.container";
+import NavigationContainer from "../container/Navigation.container";
 
 import "./app-layout.scss";
 
 const tokenCookie = getAuthTokenCookie();
 
 const AppLayout = () => {
+
+    let nav = NAVIGATION_ITEMS.find(item => `/${item.tag}` === window.location.pathname);
```

Comment:
You should be able to leverage the react-router to check the current path. I am just not certain of the exact syntax but I think they have a hook to leverage from.